docs/citing.md

```diff
@@ -0,0 +1,30 @@
+Citing a Dandiset
+-----------------
+
+If you use a Dandiset in your research, please acknowledge the Dandiset by citing it, just as you would a publication,
+including the DOI.
+The DOI can be found in the Dandiset's landing page on the DANDI Archive website.
+An example formatted citation can also be found on the Dandiset's landing page at the "CITE AS" button. This citation
+uses the DataCite citation style, which is a widely accepted standard for citing datasets, but you may need to adapt it
+to the citation style required by the journal you are submitting to.
+
+If the Dandiset has an associated publication, it may also be appropriate to cite the publication, but this does not
+replace the need to cite the Dandiset itself.
```

Comment:
```suggestion
**If the Dandiset has an associated publication, it may also be appropriate to cite the publication, but this does not
replace the need to cite the Dandiset itself.**
```